lib/ex_unit/lib/ex_unit/runner.ex

```diff
@@ -161,11 +161,22 @@ defmodule ExUnit.Runner do
     running
   end
 
-  defp spawn_modules(config, [{module, params} | modules], async?, running) do
+  defp spawn_modules(
+         config,
+         [{_group, group_modules} | modules],
```

Comment:
The sync modules are not arriving in this shape and they are being skipped (because the group modules is treated as an empty map. I will add a guard to help us catch it.

Reply:
Damn, that masked the other issues!
Thanks for fixing it.

And bummer about the tests being flaky, it would have been nice to have tests for this functionality.